nexus_constructor/nexus_filewriter_json/writer.py

```diff
@@ -204,3 +206,56 @@ def create_writer_commands(
         stop_cmd["stop_time"] = stop_time
 
     return write_cmd, stop_cmd
+
+
+def generate_forwarder_command(
+    file, streams: Dict[str, Dict[str, Any]], provider_type: str
```

Comment:
`file` is a keyword, don't use it for variable name! Maybe `output_file`? Also needs type hint.
Doc comment with `params` description would also be useful.